src/components/picker/picker.tsx

```diff
@@ -24,12 +24,14 @@ import {
     TAB_KEY_CODE,
 } from '../../util/keycodes';
 import { createRandomString } from '../../util/random-string';
+import { ActionScrollBehaviour, ActionPosition } from './actions.types';
 
 const SEARCH_DEBOUNCE = 500;
 const CHIP_SET_TAG_NAME = 'limel-chip-set';
 
 /**
  * @exampleComponent limel-example-picker
+ * @exampleComponent limel-example-picker-static-actions
```

Comment:
This is a pretty advanced use case, so I think the example should but further down. Looking at the existing examples, I think this one should probably come last.

It's ok to keep it here while working on the PR of course, but we should move it before we merge.

Reply:
I fixed it now so we don't forget later